### ISSUE TITLE
QtWebEngine not working on Arch Linux

### ISSUE THREAD

**Issue by Pan6ora** [LCA-ActivityBrowser/activity-browser#946]
Parts of AB that relay on QWebEngine (the Welcome page and Sankey diagrams) doesn't show up on my computer.

This is the same behavior as in #522 but for an Arch Linux system.

## Fix

The option `--disable-gpu` that worked for the osx problem doesn't work in my case.

I was able to fix the problem by adding the `--no-sandbox` option.

## And now ?

I don't now what causes the bug to happen. Is it only on my specific config ? On every Arch Linux ? On other Linux OS ?

The fix could look like below, but the condition will depend of the answer to previous questions.

./_activity_browser/\_\_init\_\_.py_
```
if QSysInfo.productType() == "arch":
    os.environ["QTWEBENGINE_CHROMIUM_FLAGS"] = "--no-sandbox"
    print("Info: QtWebEngine sandbox disabled")
```

Until we will now on which conditions the problem appears it may be better to just document the solution instead of pushing a fix.

**Comment by Zoophobus** [comment]
Hi Pan6ora, what version of Arch are you on? Another user has reported a similar issue by mail to me (a dependency issue with pyQt in the latest version of Ubuntu) and I'm wondering if this is the same problem?

**Comment by Pan6ora** [comment]
Hi @Zoophobus,
Arch is a rolling release system so there is no real "version" of it.
I tried with pyside2 conda versions `5.15.5` (AB version defined in meta.yml) and `5.15.8` (last version), and at different points of AB development (master branch, 2.6.9 version). None of these worked.

I remember having the same problem around March 2022, but I forgot how it ended up (somehow the problem was solved but I don't remember how).

Did the Ubuntu user solved it ? If he did I can try his solution, or in the other case he can try mine.
